packages/service-library/tests/aiohttp/test_requests_validation.py

```diff
@@ -0,0 +1,252 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+import json
+from typing import Callable
+from uuid import UUID
+
+import pytest
+from aiohttp import web
+from aiohttp.test_utils import TestClient
+from faker import Faker
+from pydantic import BaseModel, Extra, Field
+from servicelib.aiohttp.requests_validation import (
+    parse_request_body_as,
+    parse_request_path_parameters_as,
+    parse_request_query_parameters_as,
+)
+from servicelib.json_serialization import json_dumps
+
+# HELPERS -----------------------------------------------------------
+RQT_USERID_KEY = f"{__name__}.user_id"
+APP_SECRET_KEY = f"{__name__}.secret"
+
+
+def jsonable_encoder(data):
+    # q&d replacement for fastapi.encoders.jsonable_encoder
+    return json.loads(json_dumps(data))
```

Comment:
Q: this one is for use in aiohttp app only correct?

Reply:
this one is just for the test ... servicelib has no dependency to models_library where that function exists so I did a q&d=quick & dirty substitution :-)